Mathlib/SetTheory/Ordinal/Notation.lean

```diff
@@ -3,25 +3,22 @@ Copyright (c) 2018 Mario Carneiro. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro
 -/
-import Mathlib.Algebra.Ring.Divisibility.Basic
 import Mathlib.Data.Ordering.Lemmas
+import Mathlib.Data.PNat.Basic
 import Mathlib.SetTheory.Ordinal.Principal
 import Mathlib.Tactic.NormNum
-import Mathlib.Data.PNat.Basic
 
 /-!
 # Ordinal notation
 
 Constructive ordinal arithmetic for ordinals below `ε₀`.
 
 We define a type `ONote`, with constructors `0 : ONote` and `ONote.oadd e n a` representing
-`ω ^ e * n + a`.
-We say that `o` is in Cantor normal form - `ONote.NF o` - if either `o = 0` or
+`ω ^ e * n + a`. We say that `o` is in Cantor normal form - `ONote.NF o` - if either `o = 0` or
 `o = ω ^ e * n + a` with `a < ω ^ e` and `a` in Cantor normal form.
 
-The type `NONote` is the type of ordinals below `ε₀` in Cantor normal form.
-Various operations (addition, subtraction, multiplication, power function)
-are defined on `ONote` and `NONote`.
+The type `NONote` is the type of ordinals below `ε₀` in Cantor normal form. Various operations
+(addition, subtraction, multiplication, exponentiation) are defined on `ONote` and `NONote`.
```

Comment:
It looks like the original comments used https://sembr.org/

Reply:
Makes sense, I've reverted the line wrapping.